_posts/2018-04-10-programmer-misconceptions-about-art.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Programmer Misconceptions about Art
+date: 2018-04-10
+author: ash
+categories: [best practices, data, programming]
+---
+
+Our mission at Artsy has been to make art as popular as music, and at a high level, the way that our engineering team supports that mission is through building software. We have built systems and databases and user interfaces that represent different facets of the art world, and throughout our work, we have... made some mistakes.
+
+That's okay! Programmers make mistakes all the time. There is [a large list of blog posts][falsehoods] describing various programmer misconceptions, from subjects you might expect would be simple to model in computers, like units of measurement and time, to subjects that are based more in the human condition, like postal addresses and marriage.
+
+In the interest of openness and sharing what we've learned, the Artsy Engineering team has come up with the following list of misconceptions programmers believe about art.
+
+<!-- more -->
+
+- All artworks have an artist (some artworks are attributed to "cultural makers", others have a manufacturer).
+- All artworks have exactly one artist (some artworks are collaborations).
+- All artworks are unique (there are editions, reproductions, and series of works, and modeling the relationships between them all is nontrivial).
+- All lots in an art auction are artworks (some lots are "experiential", like a visit to an artist's studio).
+- Only rich people buy art.
+- Only rich people can afford to buy art, and everyone else just buys posters of "real" art.
+- All artworks have a title (some are untitled).
+- "Untitled" signifies an artwork has no title (some artworks are titled "Untitled").
```

Comment:
One more about titles…

- 512 characters should be enough to hold a title (Ummmm https://www.artsy.net/artwork/matt-goerzen-sockpuppet-theatre-representing-the-techniques-tools-and-environments-whereby-hackers-and-other-info-warriors-might-seek-to-parse-through-elsewhere-distorted-informational-domains-to-make-sense-of-them-and-also-possibly-to-acquire-by-illicit-or-clever-means-good-information-that-can-then-be-communicated-in-a-way-that-sheds-light-on-deceptions-but-can-also-be-difficult-to-evaluate-on-their-own-terms-due-to-the-elite-requisites-of-interpreting-such-knowledge-or-more-generalized-uncertaintities-regarding)